README.md

```diff
@@ -2,63 +2,101 @@
 
-| **Documentation** | **Coverage** | **[ARTENOLIS](http://opencobra.github.io/artenolis)** |
+| **Documentation** | **Test Coverage** | **[ARTENOLIS](http://opencobra.github.io/artenolis)** |
 |:-----------------:|:------------:|:--------------------------:|
 | [![doc](https://img.shields.io/badge/doc-GigaSOM-blue)](http://git.io/GigaSOM.jl) | [![coverage status](http://codecov.io/github/LCSB-BioCore/GigaSOM.jl/coverage.svg?branch=master)](http://codecov.io/github/LCSB-BioCore/GigaSOM.jl?branch=master) | [![linux](https://prince.lcsb.uni.lu/jenkins/job/GigaSOM.jl-branches-auto-linux/badge/icon)](https://prince.lcsb.uni.lu/jenkins/job/GigaSOM.jl-branches-auto-linux/) |
 
 # How to get started
 
-## Prerequisites
+## Prerequisites and requirements
 
-- Use Linux (Debian and centOS), macOS X, or Windows 10 as your operating system. `GigaSOM` has been tested on these systems.
-- In order to use `GigaSOM`, you need to install Julia 1.0 or higher. You can find the download and installation instructions for Julia [here](https://julialang.org/downloads/).
+- **Operating system**: Use Linux (Debian, Ubuntu or centOS), MacOS, or Windows 10 as your operating system. GigaSOM has been tested on these systems.
+- **Julia language**: In order to use GigaSOM, you need to install Julia 1.0 or higher. You can find the download and installation instructions for Julia [here](https://julialang.org/downloads/).
+- **Hardware requirements**: GigaSOM runs on any hardware that can run Julia, and can easily use resources from multiple computers interconnected by network. For processing large datasets, you require to ensure that the total amount of available RAM on all involved computers is larger than the data size.
 
-Once `julia` has been installed, follow [these
-instructions](https://docs.julialang.org/en/v1/manual/getting-started/) to get
-started.
-
-:bulb: If you are a complete beginner with Julia, it is advised that you familiarize youself
-with the [full documentation](https://docs.julialang.org). You should also have a
-closer look at the [Julia package manager](https://julialang.github.io/Pkg.jl/v1/getting-started/).
+:bulb: If you are new to Julia, it is adviseable to [familiarize youself with
+the environment
+first](https://docs.julialang.org/en/v1/manual/getting-started/).  Use the full
+Julia [documentation](https://docs.julialang.org) to solve various possible
+language-related problems, and the [Julia package manager
+docs](https://julialang.github.io/Pkg.jl/v1/getting-started/) to solve
+installation-related difficulties.
 
 ## Installation
 
-At the Julia prompt, add the `GigaSOM` package:
+Using the Julia package manager to install GigaSOM is easy -- after starting Julia, type:
 
 ```julia
-julia> import Pkg; Pkg.add("GigaSOM");
+import Pkg; Pkg.add("GigaSOM");
```

Comment:
leaving out `julia>` is quite confusing for newcomers. I would put that back.

Reply:
I actually like these much more (easier to copypaste, which is user-relevant as well). There was time when code snippets were highlighted with the language tag for this purpose, sadly gone. :]

Reply:
okay, maybe we can add a note...